scripts/deno/surface-lighthouse-results.ts

```diff
@@ -71,7 +67,7 @@ const results: AssertionResult[] = JSON.parse(
 /* -- Definitions -- */
 
 /** The string to search for when looking for a comment */
-const REPORT_TITLE = '⚡️ Lighthouse report';
+const IDENTIFIER_COMMENT = `<-- url: ${Deno.env.get('LHCI_URL')} -->`;
```

Comment:
Guessing the reason the comment isn't working is this?

```suggestion
const IDENTIFIER_COMMENT = `<!-- url: ${Deno.env.get('LHCI_URL')} -->`;
```

Reply:
👍 Thanks, fixed now :)